### PR TITLE
select multi days on session search filter

### LINKED ISSUE #720: Allow multiple dates to be selected in search filter
**Idea Description**
If you have selected Day1 and you select Day2, the filter will be Day2.
I would like to select Day1 or Day2.

**Reference images and links**
image

### PR DESCRIPTION
**Pull request by ked4ma**
## Issue
- close #720 

## Overview (Required)
- enable to select multi days on session search filter
- behavior and implementation are similar with category filter

## Links
- Nothing

## Screenshot
Before | After
:--: | :--:
<video src="https://user-images.githubusercontent.com/74723074/192092560-c97ba6dd-0e0f-49a3-920c-52bb1554a560.mp4" type="video/mp4" /> | <video src="https://user-images.githubusercontent.com/74723074/192092563-d5651b63-23a7-40c6-b357-91cd88d6395b.mp4" type="video/mp4" />

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SearchViewModel.kt [hunk 1]
```diff
@@ -99,15 +99,16 @@ class SearchViewModel @Inject constructor(
...
-                null
-            } else {
-                day
-            }
+            days = selectedDays.apply {
+                if (isSelected)
+                    add(day)
+                else
+                    remove(day)
+            }.sortedBy(DroidKaigi2022Day::start)
```
Comment (ked4ma) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SearchViewModel.kt:110]: sorting list here to display filtered days as sorted.
Screen Shot 2022-09-24 at 19 19 36

**Review by takahirom — APPROVED** [review]
Thanks for clean code! Looks very straightforward!

**Comment by takahirom** [comment]
Is this pr draft? Is there something you want to fix?

**Comment by ked4ma** [comment]
@takahirom 
Excuse me. I just forgot to click "ready for review".
I change status of this PR :)

**Comment by takahirom** [comment]
Thanks!

**Pull request merged**